src/main/java/com/example/spring/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package com.example.spring.controller;
+
+import com.example.spring.dto.BookDto;
+import com.example.spring.dto.CreateBookRequestDto;
+import com.example.spring.services.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/api/books")
```

Comment:
```suggestion
@RequestMapping("/books")
```

add `server.servlet.context-path=/api` to application.properties